Services/Init/classes/class.ilInitialisation.php

```diff
@@ -1462,7 +1513,7 @@ private static function initGlobalScreen(\ILIAS\DI\Container $c): void
         };
         $c->globalScreen()->tool()->context()->stack()->clear();
         $c->globalScreen()->tool()->context()->claim()->main();
-//        $c->globalScreen()->tool()->context()->current()->addAdditionalData('DEVMODE', (bool) DEVMODE);
+        //        $c->globalScreen()->tool()->context()->current()->addAdditionalData('DEVMODE', (bool) DEVMODE);
```

Comment:
We should delete this line, we don't need comments, we have the Git history :-).